app/assets/javascripts/room.js

```diff
@@ -347,3 +347,16 @@ function checkIfAutoJoin() {
     $("#room-join").click()
   }
 }
+
+function filterRooms() {
```

Comment:
Might be kind of picky on my part - but I would prefer if this was written using the jQuery syntax, since that is how most of the application's javascript is written